collectors/systemd-journal.plugin/README.md

```diff
@@ -378,31 +395,245 @@ nano /etc/systemd/system/systemd-journal-remote.service
+   when you are signing the client certificates.
+
+```sh
+SERVER_CN="myserver.example.com"
+SERVER_IP="Server_IP"
+SERVER_DNS="myserver.example.com"
+
+CLIENT_CNS=("client1.example.com" "client2.example.com")
+CLIENT_IPS=("client1_ip" "client2_ip")
+CLIENT_DNES=("client1.example.com" "client2.example.com")
+```
+
+4. Create the self-signed certificates for the server and the clients
+
+
+```sh
+openssl req -newkey rsa:2048 -nodes -out $SERVER_CN.csr -keyout $SERVER_CN.key -subj "/CN=$SERVER_CN/"
+echo "subjectAltName = IP:$SERVER_IP, DNS:$SERVER_DNS" > $SERVER_CN.ext
+openssl ca -batch -config ca.conf -notext -in $SERVER_CN.csr -out $SERVER_CN.pem -extfile $SERVER_CN.ext
+
+for i in "${!CLIENT_CNS[@]}"; do
+      CLIENT_CN="${CLIENT_CNS[$i]}"
+      CLIENT_IP="${CLIENT_IPS[$i]}"
+      CLIENT_DNS="${CLIENT_DNES[$i]}"
+      # Generate the client CSR
+      openssl req -newkey rsa:2048 -nodes -out $CLIENT_CN.csr -keyout $CLIENT_CN.key -subj "/CN=$CLIENT_CN/"
+      echo "subjectAltName = IP:$CLIENT_IP" > $CLIENT_CN.ext
+
+      # Sign the client certificate using the CA configuration
+      openssl ca -batch -config ca.conf -notext -in $CLIENT_CN.csr -out $CLIENT_CN.pem -extfile $CLIENT_CN.ext
+done
+```
+
+Keep in mind we have already produced the client certificates, we will make use of them when we will configure the clients.
+
+5. Copy the keys and the certificates into the `systemd-journal-remote`'s predefined places
+
+
+```sh
+sudo mkdir /etc/ssl/private # make sure that you havent created this folder, and use it already, you may dont want to change it's permissions
+sudo chmod 755 /etc/ssl/private
+sudo mkdir /etc/ssl/ca/ # make sure that you havent created this folder, and use it already, you may dont want to change it's permissions
+sudo chmod 755 /etc/ssl/ca
+sudo cp "${SERVER_CN}".key /etc/ssl/private/journal-remote.key # This is not predefined but we need to clarify the key
+sudo cp "${SERVER_CN}".pem /etc/ssl/certs/journal-remote.pem
+sudo cp ca.pem /etc/ssl/ca/trusted.pem
+```
+
+6. Adjust the permissions for the `systemd-journal-remote` to access them.
+
+```sh
+sudo chgrp systemd-journal-remote /etc/ssl/private/journal-remote.key
+sudo chgrp systemd-journal-remote /etc/ssl/certs/journal-remote.pem
+sudo chgrp systemd-journal-remote /etc/ssl/ca/trusted.pem
+
+
+sudo chmod 0640 /etc/ssl/private/journal-remote.key
+sudo chmod 755 /etc/ssl/certs/journal-remote.pem
+sudo chmod 755 /etc/ssl/ca/trusted.pem
+```
+
+6. Edit the `systemd-journal-remote.conf` to change the predefined key place and enable SSL.
+
+
+sh```
+sudo nano /etc/systemd/journal-remote.conf
+```
+
+You need to transform the corresponding section to something like this
+```
+[Remote]
+Seal=false
+SplitMode=host
+ServerKeyFile=/etc/ssl/private/journal-remote.key
+ServerCertificateFile=/etc/ssl/certs/journal-remote.pem
+TrustedCertificateFile=/etc/ssl/ca/trusted.pem
+```
```

Comment:
same, add 3 spaces (or even a tab would do it, exact indentation here is 3 spaces) to the text inside each list item to indent it properly, it helps with readability